### DISCUSSION TITLE
Authentication not working with IPv6 address

### DISCUSSION THREAD

**Discussion by Cilenco** [outline/outline#6410] [Self Hosting]
I'm not sure if this is a problem with my setup or with the container.

I'm hosting Outline and Authelia on my own server through Docker (both running on the same server). My domain only has an AAAA record and is therefore only raechable from IPv6 network. The login in with Authelia works fine but when I get redirected to Outline I get following error:

> {"error":"getaddrinfo ENOTFOUND auth.domain.tld","level":"error","message":"Error during authentication","stack":"Error: getaddrinfo ENOTFOUND auth.domain.tld\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:118:26)"}

I also tried to add extra_hosts to the Outline container like this:

```
extra_hosts:
  - auth.domain.tld:<Docker IPv4 of Authelia container>
```

but then I get:

> {"error":"connect ECONNREFUSED 172.24.0.4:443","level":"error","message":"Error during authentication","stack":"Error: connect ECONNREFUSED 172.24.0.4:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1595:16)"}

Do you have any ideas how to fix this?

**Comment by tommoor** [comment]
Unlikely to be ipv6 related, search the discussions for DNS resolution – there are many similar threads where the docker container is unable to resolve the IP address of your local domain.
